### ISSUE TITLE
.Net 2.0 version based on LinqBridge

### ISSUE THREAD

**Issue by GoogleCodeExporter** [blyzer/morelinq#57]
```
Currently MoreLinq references System.Core and requires .Net 3.5 to run but it 
should not be a big problem to compile it targeting 2.0 and using LinqBridge.

Target audience is probably not too big but effort to make it work should be 
relatively small either.

Thoughts?
```

Original issue reported on code.google.com by `Konst...@gmail.com` on 28 Jan 2010 at 8:57

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 3 May 2013 at 5:29
- Changed state: **Accepted**
- Added labels: **Component-Scripts**, **Milestone-Release1.0**, **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
This will only be available via the source packages for embedding in C# 
projects[1]. If someone needs a library version for .NET Framework 2.0, they 
can create a class library from the MoreLinq.Source.MoreEnumerable[1] package.

The following operators are not being back-ported:
DistinctBy, ExceptBy, ToDataTable and ToHashSet.

[1] http://www.nuget.org/packages/MoreLinq.Source.MoreEnumerable/

```

Original comment by `azizatif` on 6 May 2013 at 4:26
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 55f01290f747.
```

Original comment by `azizatif` on 6 May 2013 at 4:27
- Changed state: **Fixed**
